setup.py

```diff
@@ -4,16 +4,41 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+from os import path
+
 from setuptools import setup
 
+here = path.abspath(path.dirname(__file__))
+# Get the long description from the README file
+with open(path.join(here, "README.md")) as in_file:
+    long_description = in_file.read()
+
 setup(
-    name="gyp",
-    version="0.1",
-    description="Generate Your Projects",
-    author="Chromium Authors",
-    author_email="chromium-dev@googlegroups.com",
-    url="http://code.google.com/p/gyp",
+    name="gyp-next",
+    version="0.1.1",
+    description="A fork of the GYP build system for use in the Node.js projects",
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    author="Node.js contributors",
+    author_email="ryzokuken@disroot.org",
+    url="https://github.com/nodejs/gyp-next",
     package_dir={"": "pylib"},
     packages=["gyp", "gyp.generator"],
     entry_points={"console_scripts": ["gyp=gyp:script_main"]},
+    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*",
+    classifiers=[
+        'Development Status :: 3 - Alpha',
+        'Environment :: Console',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: BSD License',
+        'Natural Language :: English',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
```

Comment:
Do we really need these two lines?  ;-)

Reply:
I copied them, but I have no clue. Feel free to remove anything you deem unnecessary in a PR, I'd +1 that.